src/main/java/Todo.java

```diff
@@ -11,7 +11,7 @@ public Todo(String name) {
     }
 
     /**
-     * Prints task in a human-readable format.
+     * Returns task in a human-readable format.
      * @return [Type of Task][Completion Status] [Name of Task]
      */
     public String printTask() {
```

Comment:
Overall， I think you did a great job in code standards!!!: :)

Reply:
Thank you very much :)